Entities/Enemy.gd

```diff
@@ -249,6 +251,10 @@ func _set_time_type(value):
 	time_type = value
 	emit_changed()
 	
+func _set_custom_time(value):
+	custom_time = value
```

Comment:
The time string should be validated to ensure that it's in a format that the server can read. If it isn't, the user should be warned

Reply:
How about a label in yellow (add the ⚠️ emoji for good measure) that's only visible when invalid

Reply:
If the user gets feedback that it's invalid (it getting reset) it's fine by me